app/models/track.rb

```diff
@@ -29,8 +29,6 @@ def to_param
 
   memoize
   def git
-    # TODO: Slug can be removed from this
-    # once we're out of the monorepo
```

Comment:
The `slug` actually couldn't be removed after all, as it is passed to the `Git::Track` class, which itself passes it to the `Git::Repository` class.